src/flavors/mini-pwa.scss

```diff
@@ -457,10 +457,77 @@ th, td {
 @import '../mini/contextual';
 /*
   Custom style fixes and tweaks for contextual elements.
-*/
-.toast {
-  font-size: 0.875rem;
+  */
+  .toast {
+    font-size: 0.875rem;
+  }
+
+.modal {
+  position: fixed;
+  top: 0;
+  left: 0;
+  display: none;
+  width: 100vw;
+  height: 100vh;
+  background: rgba(0, 0, 0, 0.45);
+
+  .card {
+    margin: 0 auto;
+    max-height: 50vh;
+    width: 100vw;
+    overflow: auto;
+
+    .close {
+      position: relative;
```

Comment:
I suppose the change from `absolute` to `relative` is justified if the `.card` stays as-is (at `100vw`).